src/components/10-atoms/button/demo.js

```diff
@@ -28,12 +28,15 @@ storiesOf('Atoms/Button/Demos', module)
   .add('Feature - Button in a form', () => {
     let count = 0;
     const handleSubmit = ev => {
+      const { target } = ev;
+      const el = target.nodeName === 'AXA-BUTTON' ? target : target.parentNode;
```

Comment:
could this line perhaps be moved closer to where it's used, i.e. in line 39? Semantically grouped code is easier to read.

Reply:
yes makes sense

Reply:
done